armory/utils/config_loading.py

```diff
@@ -31,6 +31,7 @@
 from armory.art_experimental.attacks import patch
 from armory.art_experimental.attacks.sweep import SweepAttack
 from armory.datasets.generator import ArmoryDataGenerator
+from armory.data.datasets import EvalGenerator  # TODO: Remove before PR merge
```

Comment:
EvalGenerator is seemingly never reached as armory runs fine without this import, but flake8 complains about this line without this import: https://github.com/twosixlabs/armory/blob/0d15a1bb466fbe16dbd75d6e14e9a4511eaa1a9e/armory/utils/config_loading.py#L194
EvalGenerator should be completely removed from this file as suggested by https://github.com/twosixlabs/armory/pull/1836#discussion_r1054833213